cli/uninstall/uninstall.go

```diff
@@ -82,3 +83,32 @@ func UninstallProgram(program string, binDst string, headerDst string,
 	log.Infof("%s is uninstalled.", program)
 	return err
 }
+
+// GetList generates a list of options to uninstall.
+func GetList(cmdCtx *cmdcontext.CmdCtx) []string {
+	list := []string{}
+	re := regexp.MustCompile("^(?P<prog>(?:tarantool)|(?:tarantool-ee)|(?:tt))_(?P<ver>.*)$")
+
+	cliOpts, err := configure.GetCliOpts(cmdCtx.Cli.ConfigPath)
+	if err != nil {
+		return nil
+	}
+
+	if cliOpts.App.BinDir == "" {
+		return nil
+	}
+
+	installed, err := os.ReadDir(cliOpts.App.BinDir)
+	if err != nil {
+		return nil
+	}
+
+	for _, file := range installed {
+		matches := util.FindNamedMatches(re, file.Name())
+		if len(matches) != 0 {
+			list = append(list, matches["prog"]+search.VersionCliSeparator+matches["ver"])
```

Comment:
Will this work if I enter
`tt uninstall tt[tab]`
? In this case, I'd like to se only tt versions.

Reply:
Yes